app/utils/transformers/DeclarationGoodsItemNumberTransformer.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils.transformers
+
+import models.{Index, UserAnswers}
+import pages.houseConsignment.index.items.DeclarationGoodsItemNumberPage
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class DeclarationGoodsItemNumberTransformer @Inject() (implicit ec: ExecutionContext) extends PageTransformer {
```

Comment:
I don't think we get any value out of this and the `GoodsItemNumberTransformer` being separate classes. Otherwise we will end up with A LOT of these classes